github workflow/src/cakes.py

```diff
@@ -1,6 +1,8 @@
 from src.utils.cooking_techniques import mix, bake
 
 def banana_cake(*ingredients):
+    if "banana" not in ingredients:
+        print("that is so sad :(")
```

Comment:
```suggestion
        print("Missing banana for cake")
```

this is not a very helpful message